src/pages/ErrorPage/NotFoundPage.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import FullPageNotFoundView from '../../components/BlockingViews/FullPageNotFoundView';
+
+// eslint-disable-next-line rulesdir/no-negated-variables
+const NotFoundPage = () => (
```

Comment:
NAB But can we rename it?

cc: @pecanoro

Reply:
That's why I marked it as NAB

Reply:
Oh wow I didn't even know this was a lint rule lol Yes, this is the best name for the component, calling it MissingPage would be super odd since it's `not found` all around the internet